### PR TITLE
Update guide

### PR DESCRIPTION
**Pull request by L4xus**
This was for updating the venv guide

### CODE REVIEW

**Comment by L4xus** [comment]
also updated src errors because of wrong import style

File: src/modlunky2/cli.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 from flask import Flask
 
-from .code_execution import CodeExecutionManager
-from .views.assets import blueprint as assets_blueprint
-from .views.entities import blueprint as entities_blueprint
-from .views.index import blueprint as index_blueprint
+from code_execution import CodeExecutionManager
```
Comment (gmjosack) [inline src/modlunky2/cli.py:7]: Why are you changing these imports? are you encountering some kind of error? These are called relative imports and they distinguish imports within the package from top level imports.

File: README.md [hunk 2]
```diff
@@ -23,33 +23,45 @@ If you'd like to contribute to `modlunky2` here are some steps to setup your env
 ### Creating VirtualEnv
 In the root directory you can make a virtualenv. It will be excluded from commits by default
 ```console
-> python -m venv venv
+python -m venv venv
```
Comment (gmjosack) [inline README.md:26]: Why did you remove the console marker? You didn't explain anywhere why you made this change and it wasn't applied consistently as you let them in the setup section. I tend to prefer a console character to distinguish where input goes in an example.

File: README.md [hunk 3]
```diff
@@ -23,33 +23,45 @@ If you'd like to contribute to `modlunky2` here are some steps to setup your env
...
 ```
 
 ### Activate VirtualEnv
 
 You'll want to activate the virtual environment whenever you're testing any commands from this package
 
 #### Powershell
+
+please check here first https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7
```
Comment (gmjosack) [inline README.md:35]: Maybe we should just remove the whole section on how to use a virtualenv and just link to the comprehensive docs: https://docs.python.org/3/library/venv.html#creating-virtual-environments
